src/main/kotlin/com/ampnet/identityservice/persistence/model/Document.kt

```diff
@@ -5,10 +5,15 @@ import javax.persistence.Embeddable
 
 @Embeddable
 class Document(
+    // type: String(one of PASSPORT, ID_CARD, DRIVERS_LICENSE, RESIDENCE_PERMIT, OTHER) Document type
     var type: String?,
+    // country: ISO-2 String Document country
     var country: String?,
+    // number: String Document number
     var number: String?,
```

Comment:
What is the number format here, something that uses dashes (e.g. `123-456`)? Would it not be possible to use `BigInteger` here?